apps/src/componentLibrary/StylizedBaseDialog.jsx

```diff
@@ -80,6 +80,7 @@ export default function StylizedBaseDialog(props) {
       onClick={props.handleCancellation || props.handleClose}
     />,
     <FooterButton
+      className="uitest-base-dialog-confirm"
```

Comment:
Just curious, is prefixing these with "uitest-" a standard (and if not, should it be)?

Reply:
I saw this pattern in some teacher dashboard tests I was looking at -- searching through our repo, we have a lot of instances of `uitest-` (662) and `ui-test-` (481), so I think either of those works.